platforms/f2/index.ts

```diff
@@ -0,0 +1,46 @@
+import { adEngineConfigured, bootstrapAndGetConsent, PlatformStartup } from '@platforms/shared';
+import { context } from '@wikia/ad-engine';
+import { Container } from '@wikia/dependency-injection';
+import { Action, Communicator, setupPostQuecast } from '@wikia/post-quecast';
+import { take } from 'rxjs/operators';
+import { action } from 'ts-action';
+import { ofType } from 'ts-action-operators';
+import { basicContext } from './ad-context';
+import { setupF2Ioc } from './setup-f2-ioc';
+import './styles.scss';
+
+setupPostQuecast();
+
+export const f2ready = action('[F2] Configured');
```

Comment:
Consider creating config type, and include it like:

```suggestion
export const f2Ready = action('[F2] Configured', props<F2Config>());
```

Reply:
Where you think it should be exported?

Reply:
Discussed on slack.